docs/sources/mimir/references/http-api/index.md

```diff
@@ -864,6 +864,8 @@ The `file`, `rule_group` and `rule_name` parameters are optional, and can accept
 
 The `exclude_alerts` parameter is optional. If set, it only returns rules and excludes active alerts.
 
+The `group_limit` and `group_next_token` parameters are optional. If `group_limit` is set, it will limit the number of rule groups returned in a single response. If the total number of rule groups exceeds this value, the response will contain a `groupNextToken`.
```

Comment:
```suggestion
The `group_limit` and `group_next_token` parameters are optional. If `group_limit` is set, it limits the number of rule groups returned in a single response. If the total number of rule groups exceeds this value, the response contains a value of `groupNextToken`.
```

Reply:
We avoid using "will" in docs in favor of present tense.

Reply:
Should these two new parameters be added to the endpoint definition itself? I.e., `GET <prometheus-http-prefix>/api/v1/rules?type={alert|record}&file={}&rule_group={}&rule_name={}&exclude_alerts={true|false}`.